### PR TITLE
Improved manual beer meter

### PR DESCRIPTION
**Pull request by DomantasDev**

### CODE REVIEW

**Comment by augisjay** [comment]
Perhaps I would suggest to take into consideration another way to meter the amount of poured beer, like professor said, it should have not percentage, but some kind of point rating system, like 1 is "i had a fight with bartender" and 9 "spilled my beer when tried to sip" to 10 "BEER OVERFLOW". What do others think?

File: BeerOverflowWindowsApp/BeerPanel.cs [hunk 1]
```diff
@@ -77,7 +77,8 @@ private void BeerPanel_MouseMove(object sender, MouseEventArgs e)
                 if (marker.Y + marker.Height / 2 > bottom.Y)
                     marker.Y = bottom.Y - marker.Height / 2;
                 Rating = (bottom.Y - marker.Y - marker.Height / 2) / (height / 100) + "";
```
Comment (augisjay) [inline BeerOverflowWindowsApp/BeerPanel.cs]: Perhaps I would suggest to take into consideration another way to meter the amount of poured beer, like professor said, it should have not percentage, but some kind of point rating system, like 1 is "i had a fight with bartender" and 9 "spilled my beer when tried to sip" to 10 "BEER OVERFLOW". What do others think?

Comment (DomantasDev) [reply BeerOverflowWindowsApp/BeerPanel.cs]: Will do that

File: BeerOverflowWindowsApp/GetBarList.cs [hunk 2]
```diff
@@ -78,5 +78,15 @@ private void RateBarsButton_Click(object sender, EventArgs e)
             var ratingsForm = new BarRatingsForm();
             ratingsForm.Show();
         }
+
```
Comment (augisjay) [inline BeerOverflowWindowsApp/GetBarList.cs]: Why are these classes left empty?

File: BeerOverflowWindowsApp/ManualBarRating.cs [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+            for(int i = 0; i < 6; i++)
+            {
+                beerGlassList.Add(new Rectangle(i * ImageSize, 0, ImageSize, ImageSize));
+            }
+            Refresh();
+        }
+
+        protected override void OnPaint(PaintEventArgs pe)
+        {
+
```
Comment (augisjay) [inline BeerOverflowWindowsApp/ManualBarRating.cs]: Unnecessary gap.

File: BeerOverflowWindowsApp/ManualBarRating.cs [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+                }
+        }
+
+        public static Bitmap ResizeImage(Image image, int width, int height)
+        {
+            var destRect = new Rectangle(0, 0, width, height);
+            var destImage = new Bitmap(width, height);
+
+            destImage.SetResolution(image.HorizontalResolution, image.VerticalResolution);
+
```
Comment (augisjay) [inline BeerOverflowWindowsApp/ManualBarRating.cs]: Gaps

File: BeerOverflowWindowsApp/ManualBarRating.cs [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+                graphics.SmoothingMode = SmoothingMode.HighQuality;
+                graphics.PixelOffsetMode = PixelOffsetMode.HighQuality;
+
+                using (var wrapMode = new ImageAttributes())
+                {
+                    wrapMode.SetWrapMode(WrapMode.TileFlipXY);
+                    graphics.DrawImage(image, destRect, 0, 0, image.Width, image.Height, GraphicsUnit.Pixel, wrapMode);
+                }
+            }
+
```
Comment (augisjay) [inline BeerOverflowWindowsApp/ManualBarRating.cs]: Unnecessary gap.

File: BeerOverflowWindowsApp/GetBarList.cs (hunk 2 above)
Comment (DomantasDev) [reply BeerOverflowWindowsApp/GetBarList.cs]: Whoops, Designer did it automatically, I will remove it

File: BeerOverflowWindowsApp/ManualBarRating.cs (hunk 3 above)
Comment (DomantasDev) [reply BeerOverflowWindowsApp/ManualBarRating.cs]: Will remove all unnecessary gaps after I finish this class completely.

File: BeerOverflowWindowsApp/GetBarList.cs (hunk 2 above)
Comment (DomantasDev) [reply BeerOverflowWindowsApp/GetBarList.cs]: And those are methods, not classes. ^^

Comment (augisjay) [reply BeerOverflowWindowsApp/GetBarList.cs]: You're right, sorry for this

**Comment by CactusGreen** [comment]
What is needed to merge this?

**Pull request merged**